src/plugins/screenshot/Screenshot.cc

```diff
@@ -0,0 +1,276 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#include "Screenshot.hh"
+
+#include <ignition/msgs/boolean.pb.h>
+#include <ignition/msgs/stringmsg.pb.h>
```

Comment:
These `ign-msgs` includes introduce Windows warnings. We can suppress the warnings like this:

https://github.com/ignitionrobotics/ign-gui/blob/10affb2b101c333f0e4c194e79444ba9179f6b61/src/plugins/image_display/ImageDisplay.hh#L21-L28

The same goes for the test.

Reply:
Thanks! a393787385e8b469bcaae8a5d44f8481717ed24c